packages/boilerplates/ICanFindOnlineBoilerplatePackages.ts

```diff
@@ -0,0 +1,45 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Dolittle. All rights reserved.
+*  Licensed under the MIT License. See LICENSE in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { BoilerplatePackageJson } from "./internal";
+
+/**
+ * Responsible for discovering boilerplates
+ *
+ * @export
+ * @interface ICanFindOnlineBoilerplatePackages
+ */
+export interface ICanFindOnlineBoilerplatePackages {
```

Comment:
Are there multiple implementations of this and they are discovered? If not, it should probably be named differently. We're lacking guidance on naming of this - added a task to improve documentation on naming.